.circleci/config.yml

```diff
@@ -392,17 +402,15 @@ references:
       - *upload_github_release_asset
       - *upload_release_artifacts
 
-  upload_latest_artifacts: &upload_latest_artifacts
-    - checkout
-    - *set_package_path
-    - setup_secrets
-    - attach_workspace:
-        at: *packages_workspace
-    - run:
-        name: Upload build artifacts to central repository (AWS)
-        command: |
-          envdir /secrets aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/aeternity-latest-${PKG_SUFFIX:?}.tar.gz
-    - *fail_notification
+  upload_latest_package: &upload_latest_package
```

Comment:
I don't like the naming, not each package is artifact

Reply:
it was for consistency with `upload_package`

Reply:
also there is already `&upload_latest_artifacts`